pkg/apis/interconnectedcloud/v1alpha1/interconnect_types.go

```diff
@@ -105,6 +105,7 @@ type DeploymentPlanType struct {
 	Resources    corev1.ResourceRequirements `json:"resources,omitempty"`
 	Issuer       string                      `json:"issuer,omitempty"`
 	LivenessPort int32                       `json:"livenessPort,omitempty"`
+	ServiceType  string                      `json:"serviceType,omitempty"`
```

Comment:
Change needs update to:
deploy/olm-catalog/qdr-operator/0.1.0/catalog-source.yaml
pkg/apis/interconnectedcloud/v1alpha1/zz_generated.deepcopy.go

Reply:
catalog source has been regenerated.

What should trigger the deepcopy update? A full make seems not to make any change.

Reply:
Looks like the simple typed fields aren't explicitly copied, but rely on the struct level assignment, so this change I think is not expected to regenerate the method.